client/js/components/procedure/DpSubmitterList.vue

```diff
@@ -188,15 +208,24 @@ export default {
       return initialOrganisationDepartmentName || '-'
     },
 
-    handleOrgaPostalCodeAndOrgaCity (initialOrganisationCity, initialOrganisationPostalCode) {
+    handleOrgaAddress (initialOrganisationCity, initialOrganisationPostalCode, initialOrganisationStreet, initialOrganisationHouseNumber) {
+      let fullAddress = ''
+      if (initialOrganisationStreet) {
+        fullAddress = initialOrganisationHouseNumber ? initialOrganisationHouseNumber + ' ' + initialOrganisationStreet : initialOrganisationStreet
+      }
       if (initialOrganisationPostalCode) {
-        return initialOrganisationCity ? initialOrganisationPostalCode + ' ' + initialOrganisationCity : initialOrganisationPostalCode
+        fullAddress += initialOrganisationStreet ? ', ' : ''
+        fullAddress += initialOrganisationCity ? initialOrganisationPostalCode + ' ' + initialOrganisationCity : initialOrganisationPostalCode
       }
-      return initialOrganisationCity || '-'
+      return fullAddress || '-'
```

Comment:
Within the function, i suggest to drop the `initialOrganisation` from every variable name because it harms readability (also see above suggestion).

Reply:
thx, done!